components/scream/src/share/field/field_layout.cpp

```diff
@@ -135,5 +135,23 @@ LayoutType get_layout_type (const std::vector<FieldTag>& field_tags) {
   return result;
 }
 
+std::string print (const FieldLayout& layout)
```

Comment:
In Python, this function would be called `str` or `repr`.

Reply:
Actually `__str__` or `__repr__`, but the `str` and `repr` functions are what the caller would use to access it. Just another perspective.

Reply:
I opted for `to_string`, mimicking ekat's util for `Units` (and some std utils).